ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/api/BatchingQueueSpec.scala

```diff
@@ -46,7 +46,7 @@ class BatchingQueueSpec
         res <- queue.offer(correlatedSubmission)
       } yield {
         res should be(SubmissionResult.Acknowledged)
-        eventually {
+        eventually(Timeout(1.second)) {
```

Comment:
You can also override the patience configuration for the entire test class by adding this to the top:

```scala
  implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = 1.second)
```

This works too though.

Reply:
I'd go for the smaller change b/c 1. the default should really work most often and, if not, better to look into it + 2. this is going away soon anyway.

Reply:
Ship it. Just bear in mind we override this _everywhere_ because the default does not work on our CI. :grin:

Reply:
:burn: